### ISSUE TITLE
Pool temperatures show incorrectly

### ISSUE THREAD

**Issue by soupman98** [maartenvantjonger/omnilogic-smartapp#8]
Love the interface, integration and work. Thanks. My pool heater temps show extremely odd values. It' s not a celcius/Fahrenheit thing.. just a high range it looks like. My graph IDE shows the following which are obviously not at all correct. Any ideas?

heatingSetpoint: 142 F
thermostatSetpointRange: [131,190] F
coolingSetpointRange: [131,190] F
thermostatSetpoint: 142 F
thermostatMode: off
coolingSetpoint: 0
thermostatOperatingState: idle

**Comment by maartenvantjonger** [comment]
@soupman98 It appears to be converting the temperatures coming from the API to Fahrenheit despite them already being in Fahrenheit. 
I'd like to see your MSP Config for this issue as well. The temperature unit in this config is apparently something other than `UNITS_FAHRENHEIT`.

**Comment by maartenvantjonger** [comment]
Hey @soupman98 I fixed this problem in the `develop` branch. (https://github.com/maartenvantjonger/omnilogic-smartapp/tree/development)
The problem was that I wasn't checking if the sensor that is used to get the temperature unit (F vs C) was actually a temperature sensor. In your case I was getting your flow sensor, which obviously doesn't give me a temperature unit.

Updating both `omnilogic.groovy` and `omnilogic-heater.groovy`, and removing/re-adding the heater device should do the trick. Let me know if that works!

**Comment by soupman98** [comment]
@maartenvantjonger  Do I do something in Graph/Groove/IDE to map to the development branch? or just copy the content over manually? Sorry, I don't live in GitHub regularly...

**Comment by maartenvantjonger** [comment]
> @maartenvantjonger Do I do something in Graph/Groove/IDE to map to the development branch? or just copy the content over manually? Sorry, I don't live in GitHub regularly...

Yep, see https://github.com/maartenvantjonger/omnilogic-smartapp/issues/9#issuecomment-997413568
